File: solved/two_sum.php
<?php 
$nums = [3,3];
$target = 6;
$len = count($nums);
for($i = 0; $i<$len; $i++){
    for($j=$i+1; $j <$len; $j++){
        if($nums[$i]+$nums[$j] == $target){
            print_r($result = [$i,$j]);
        }
    }
}

File: practice/twosum.py

def two_sum(nums, target):
    remainder = []
    for i in range(len(nums)):
        remain = target-nums[i]
        if (nums[i] in remainder):
            return (i,remainder.index(nums[i]))
        else:
            remainder.insert(i,remain)
        
# Example usage:
nums = [2,3,4]
target = 6
result = two_sum(nums, target)
print(result)
